codice/ddf
# DDF-4319 Add descriptions to the sources in the Admin UI

##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
Link to 2.13.x PR: #4016 
#### What does this PR do?
Updates the Source selection dropdown in the Admin UI to display the metatype description for the selected source.
#### Who is reviewing it? 
@ethantmanns 
@tyler30clemens 
@paouelle 
@brjeter 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@tbatie
#### How should this be tested?
Test in multiple browsers
- Go to the Admin UI -> Catalog -> Sources tab 
- Add a source
- Verify that a description appears below the Source Type field
- Change the Source Type and verify the description changes
#### Any background context you want to provide?
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/DDF-4319)
#### Screenshots
![image](https://user-images.githubusercontent.com/17188993/48208459-12e91280-e330-11e8-99cd-d8203addfea6.png)
#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
